index.js

```diff
@@ -29,20 +29,40 @@ module.exports = (options) => {
     const socket = {
       connectTimeout: 10000,
       keepAlive: 5000,
+      reconnectStrategy: retries => {
+        logger.info('Reconnect attempt', retries);
```

Comment:
Not sure, but I think `logger.info` doesn't support over one parameter. May be safer to do this instead:

```
logger.info(`Reconnect attempt ${retries}`);
```